recipes-wpe/wpeframework/include/wpeframework.inc

```diff
@@ -9,7 +9,7 @@ PACKAGECONFIG[debug]            = "-DCMAKE_BUILD_TYPE=Debug,-DCMAKE_BUILD_TYPE=R
 
 S = "${WORKDIR}/git"
 
-inherit cmake pkgconfig
+inherit cmake pkgconfig python3native
```

Comment:
this ensures that it will always use consistent version of python. but if there is no such need then asking for `python3` in interpreter section of scripts should be ok